mycroft/util/audio_test.py

```diff
@@ -103,14 +103,21 @@ def main():
         print()
 
     config = Configuration()
+    device_index = None
     if "device_name" in config["listener"]:
         dev = config["listener"]["device_name"]
+        device_index = find_input_device(dev)
+        if not device_index:
+            raise ValueError("Device with name {} not found, check your configuration".format(dev))
```

Comment:
Should these be 'f strings'?

Reply:
I used the existing code style. It could be converted to f strings but it will not change anything.

Reply:
yes, we are slowly moving the whole codebase to f strings, if you touch a file change those :)